semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -453,9 +456,7 @@ def _sort_attributes_dict(
         :param attributes: Dictionary of attributes to sort
```

Comment:
The function docs (cut off by GH context) claim "First  imported, and then defined attributes.". I think this is not true anymore, now it's simply alphabetical? Also maybe check if some other follow-up simplifications/comment-corrections are possible now.

Reply:
good point! I simplified code a bit